Makefile

```diff
@@ -0,0 +1,54 @@
+NAME=chr-terminology-server
+GIT_VERSION=$(shell git rev-parse HEAD)
+SEMVER_VERSION=$(shell git describe --abbrev=0 --tags)
+REPO=quay.io/babylonhealth
+DEPLOY_DEV_URL=http://dev-ai-deploy.babylontech.co.uk:5199/job/kube-deploy-dev/buildWithParameters
+DEPLOY_STAGING_URL=http://dev-ai-deploy.babylontech.co.uk:5199/job/kube-deploy-staging/buildWithParameters
+SNOWOWL_RPM_PACKAGE=$(shell find ./releng/com.b2international.snowowl.server.update/target -name "snow-owl-oss*.rpm")
+
+
+build-project:
+	./mvnw clean verify -DskipTests
+
+build-docker:
+	docker build ./docker \
+	--build-arg SNOWOWL_RPM_PACKAGE=`basename "${SNOWOWL_RPM_PACKAGE}"` \
```

Comment:
what's SNOWOWL_RPM_PACKAGE used for?

Reply:
that's basically the build rpm product that is copied inside the docker, but because the name is dependant on the build version and the timestamp it has to be passed as an arg